README.md

```diff
@@ -115,6 +117,12 @@ scp -r <remote-machine>:/path/to/log/ ./torch-profs/
 tensorboard --logdir=./torch-profs/log
 ```
 
+**Setup note:** If you don't see messages indicating that runs named 'transformers' are being loaded, particularly for Windows, you may want to run the following command, or verify that the python package exists.
```

Comment:
the `torch-tb-profiler` package is needed for all users, windows or otherwise. Just addressed this in https://github.com/Quentin-Anthony/torch-profiling-tutorial/commit/1294803ab518014fe3e331ecdaff9b72b67bc9bb